.env.defaults

```diff
@@ -81,3 +81,17 @@ RHUB_SKIP_INIT=
 
 # Default logging level if LOG_CONFIG is not set.
 LOG_LEVEL=info
+
+################################################################################
```

Comment:
I think we need to add this to the deployment repositories (rhub-docker, rhub-deployment) on gitlab.

Reply:
These variables are optional and if not defined notifications will be disabled, but yeah, I'll update deployment repos.